activity/activity.go

```diff
@@ -109,3 +111,7 @@ func GetWorkerStopChannel(ctx context.Context) <-chan struct{} {
 func IsActivity(ctx context.Context) bool {
 	return internal.IsActivity(ctx)
 }
+
+func ExtraInformation() ActivityExtra {
+	return internal.NewActivityExtra()
+}
```

Comment:
Recommend:

```go
func GetExtraAttributes(ctx context.Context) ExtraAttributes {
    return internal.GetActivityExtraAttributes(ctx)
}

func SetExtraAttributes(ctx context.Context, attrs ExtraAttributes) error {
    return internal.SetActivityExtraAttributes(ctx, attrs)
}
```

There are some alternative designs, but I feel this is simplest. Also, please add Godoc to all public API. Also, please mark this as experimental for now until we are confident in the server-side implementation.